### PR TITLE
Issue #2802699 by kim.pepper: Update Default Content module to 8.x-1.0-alpha3 release

### PR DESCRIPTION
**Pull request by kimpepper**

### CODE REVIEW

**Comment by benjy** [comment]
Do we need to re-export all the content as part of this?

File: composer.json [hunk 1]
```diff
@@ -48,7 +48,8 @@
   "require-dev": {
     "phing/phing": "~2.11",
     "previousnext/phing-phpqatools": "dev-master",
-    "drush/drush": "~8.1.0"
+    "drush/drush": "~8.1",
+    "symfony/dependency-injection": "~2"
```
Comment (benjy) [inline composer.json]: Why is this needed? Shouldn't drush or core bring that in as they need it?

**Comment by benjy** [comment]
Rest looks good to me.

File: composer.json (hunk 1 above)
Comment (kimpepper) [inline composer.json]: There was an issue where drush was bringing in symfony/dependency-injection: ~3 which was causing a `Error: Undefined class constant 'Symfony\Component\DependencyInjection\ContainerInterface::SCOPE_CONTAINER`
